UnitsNet.Tests/CustomCode/MassFlowTests.cs

```diff
@@ -31,7 +31,7 @@ public class MassFlowTests : MassFlowTestsBase
         protected override double DecagramsPerSecondInOneGramPerSecond => 1E-1;
 
         protected override double HectogramsPerSecondInOneGramPerSecond => 1E-2;
-        protected override double KilogramsPerHourInOneGramPerSecond => 3.6;
+        protected override double KilogramsPerHourInOneGramPerSecond => 0.277778;
```

Comment:
Uh-oh :-) You just flipped the implementation upside down so it is no longer correct. This is very easy to do, so my go-to sanity check is to google `KilogramsPerHour In OneGramPerSecond` and that tells me this should read `3.6`.